queries/rst/highlights.scm

```diff
@@ -158,6 +158,17 @@
 (title) @text.title
 
 (comment) @comment
+(comment) @spell
 (comment "..") @comment
 
+(directive
+    name: (type) @directive
```

Comment:
`@directive` is not in the allowed list

Reply:
@stsewd

Reply:
That's fine, I can underscore.

Reply:
If you don't want it highlighted, that's fine.

Reply:
Nah, it's just a handle to pass to the predicate.